### PR TITLE
Rei 04 - Implement module and pages navigation.

### PR DESCRIPTION
**Pull request by NelsonChad**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt [hunk 1]
```diff
@@ -12,40 +14,61 @@ import androidx.compose.runtime.Composable
...
 @Composable
-fun MenuScreen() {
+fun MenuScreen(context: Context) {
     Column(
         modifier = Modifier
             .fillMaxWidth()
             .height(170.dp)
-            .background(color = Color(0xFFFFFFFF))
-            .padding(5.dp),
+            .background(color = Color(0xFFFFFFFF)),
```
Comment (CarlosMacaneta) [inline support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: set color in colors file

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt [hunk 2]
```diff
@@ -12,40 +14,61 @@ import androidx.compose.runtime.Composable
...
             verticalAlignment = Alignment.CenterVertically
         ) {
             // First Card
-            CustomCard(imageResId = R.drawable.manual_icon, title = R.string.manuals.toString())
+            CustomCard(
+                imageResId = R.drawable.manual_icon,
+                title = stringResource(id = R.string.manuals),
+                onClick = {
+                    val intent = Intent(context, MainActivity::class.java)
+                    intent.putExtra("screen", Screen.Manuals.route)
```
Comment (CarlosMacaneta) [inline support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: create constant file and sent key constants
```suggestion
                    intent.putExtra("screen", Screen.Manuals.route)
                    intent.putExtra("screen", Screen.Support.route)
```

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt [hunk 3]
```diff
@@ -12,40 +14,61 @@ import androidx.compose.runtime.Composable
...
+            )
         }
     }
 }
 
 @Preview(showBackground = true)
 @Composable
 fun MyScreenPreview() {
-    MenuScreen()
+    //MenuScreen()
```
Comment (CarlosMacaneta) [inline support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: remove this section

File: support-module/src/main/java/com/saudigitus/support_module/utils/AppNavHost.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    NavHost(
+        navController = navController,
+        startDestination = route,
+    ) {
+        composable(Screen.Menu.route) {
+            MenuScreen(context = LocalContext.current)
+        }
+        composable(Screen.Manuals.route) {
+            ManualScreen(navController = navController, onBack = {
+                Timber.d("Back button clicked 01")
```
Comment (CarlosMacaneta) [inline support-module/src/main/java/com/saudigitus/support_module/utils/AppNavHost.kt]: remove timber...

File: support-module/src/main/java/com/saudigitus/support_module/utils/AppNavHost.kt [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                activity.finish()
+            })
+        }
+        composable(Screen.Support.route) {
+            SupportScreen(navController = navController, onBack = {
+                Timber.d("Back button clicked 02")
```
Comment (CarlosMacaneta) [inline support-module/src/main/java/com/saudigitus/support_module/utils/AppNavHost.kt]: remove timber...

**Review by CarlosMacaneta — CHANGES_REQUESTED** [review]

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt (hunk 1 above)
Comment (NelsonChad) [reply support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: Done, the color is White

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt (hunk 2 above)
Comment (NelsonChad) [reply support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: Done

File: support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt (hunk 3 above)
Comment (NelsonChad) [reply support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt]: Fixed this point

**Review by CarlosMacaneta — APPROVED** [review]

**Pull request merged**